pkg/controllers/provisioning/scheduling/scheduler.go

```diff
@@ -59,34 +65,67 @@ func NewScheduler(kubeClient client.Client) *Scheduler {
 func (s *Scheduler) Solve(ctx context.Context, provisioner *v1alpha5.Provisioner, instanceTypes []cloudprovider.InstanceType, pods []*v1.Pod) ([]*Node, error) {
 	defer metrics.Measure(schedulingDuration.WithLabelValues(injection.GetNamespacedName(ctx).Name))()
 	constraints := provisioner.Spec.Constraints.DeepCopy()
+	start := time.Now()
```

Comment:
This time is already being measured by line 65.

Reply:
That data isn't readily available without running prometheus though, right?  It's measured, but I wanted to know how long scheduling was taking and normally just tail the controller logs.